.gitignore

```
@@ -0,0 +1,13 @@
+.DS_Store
+.idea
+
+.ddev/*
+!.ddev/config.yaml
+!.ddev/docker-compose.environment.yaml
+
+/public/*
+!/public/typo3conf
```

```suggestion
!/public/.htaccess
!/public/typo3conf
```
I always forget to add the `.htaccess` file. But without, the Slugs won't work. Please have look, if you have a `.htaccess` file in your `/public/`.